### PR TITLE
docs(samples): add Video Stitcher slate and CDN key code samples and tests

### PR DESCRIPTION
**Pull request by irataxy**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-video-stitcher/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes b:202972969

### CODE REVIEW

File: samples/listCdnKeys.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  // location = 'us-central1';
+
+  // Imports the Video Stitcher library
+  const {VideoStitcherServiceClient} =
+    require('@google-cloud/video-stitcher').v1;
+  // Instantiates a client
+  const stitcherClient = new VideoStitcherServiceClient();
+
+  async function listCdnKeys() {
+    const [cdnKeys] = await stitcherClient.listCdnKeys({
```
Comment (alexander-fenster) [inline samples/listCdnKeys.js]: This method supports automatic pagination, and will keep requesting keys until the whole list is retrieved, and might hit the request quota. We recommend using async iteration (`listCdnKeysAsync`) as described [here](https://github.com/googleapis/gax-nodejs/blob/main/client-libraries.md#auto-pagination). Same applies to other paginated methods (`listVodStitchDetails`, `listVodAdTagDetails`, `listLiveAdTagDetails`, `listSlates`).

**Comment by sofisl** [comment]
@irataxy, would you mind adding your team in the .repo-metadata.json file as codeowners as well?

**Review by alexander-fenster — APPROVED** [review]
Please consider changing list methods to their async iterator versions.

File: samples/listCdnKeys.js (hunk 1 above)
Comment (irataxy) [reply samples/listCdnKeys.js]: Done

**Pull request merged**